src/seedu/addressbook/AddressBook.java

```diff
@@ -526,6 +537,62 @@ private static boolean isDeletePersonArgsValid(String rawArgs) {
             return false;
         }
     }
+    
+    /**
+     * Edits the phone number of a person identified using last displayed index.
+     *
+     * @param commandArgs full command args string from the user
```

Comment:
This should be more descriptive, maybe state what the command args should contain?